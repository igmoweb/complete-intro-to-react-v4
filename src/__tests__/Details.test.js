import React from "react";
import { create } from "react-test-renderer";
import Details from "../Details";

test("Snapshot", () => {
  const c = create(<Details />);
  expect(c.toJSON()).toMatchSnapshot();
});

test("Show modal", () => {
	const c = create(<Details search={() =>{}}/>);
	const instance = c.getInstance();
	expect( instance.state.showModal ).toBe( false );
	instance.toggleModal();
	expect( instance.state.showModal ).toBe( true );
})
