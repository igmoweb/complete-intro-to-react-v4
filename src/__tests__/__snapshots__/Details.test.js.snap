// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<h1>
  loading â€¦ 
</h1>
`;
